modules/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConnectionManager.java

```diff
@@ -64,13 +54,9 @@
 public class HttpConnectionManager {
 
     private static HttpConnectionManager instance = new HttpConnectionManager();
-    private Map<String, org.wso2.transport.http.netty.contract.ServerConnector>
-            startupDelayedHTTPServerConnectors = new HashMap<>();
-
-    private Map<String, org.wso2.transport.http.netty.contract.ServerConnector>
-            startedHTTPServerConnectors = new HashMap<>();
-    private Map<String, HttpServerConnectorContext>
-            serverConnectorPool = new HashMap<>();
+    private Map<String, ServerConnector> startupDelayedHTTPServerConnectors = new HashMap<>();
+    private Map<String, ServerConnector> startedHTTPServerConnectors = new HashMap<>();
```

Comment:
What is the purpose of this map. Map content is never used

Reply:
Removed this map. I think it was a remaining piece of code from an older implementation